src/Error/JsonApiExceptionRenderer.php

```diff
@@ -97,6 +97,10 @@ public function validation($exception)
         // set data and send response
         $this->controller->response->type('jsonapi');
         $this->controller->response->body($json);
+        $this->controller->response->cors($this->controller->request)
```

Comment:
It's upto you to set appropriate CORS headers for your app. The plugin won't do it for you.